docker/php-cli/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM php:7.4.15-cli
```

Comment:
Pretty simple really. A standard Docker PHP CLI image with Composer (Git/Zip/7Zip) and the default development configuration enabled, which should be sufficient.